compilers/concrete-optimizer/concrete-optimizer/src/optimization/dag/multi_parameters/operations_value.rs

```diff
@@ -0,0 +1,179 @@
+use std::ops::{Deref, DerefMut};
+
+/**
+ * Index actual operations (input, ks, pbs, fks, modulus switching, etc).
+ */
+#[derive(Clone, Copy, Debug, Eq, PartialEq, PartialOrd)]
+pub struct Indexing {
+    /* Values order
+    [
+        // Partition 1
+        // related only to the partition
+        fresh, pbs, modulus,
+        // Keyswitchs to small, from any partition to 1
+        ks from 1, ks from 2, ...
+        // Keyswitch to big, from any partition to 1
+        ks from 1, ks from 2, ...
+
+        // Partition 2
+        // same
+    ]
```

Comment:
Couldn't we use a mor complex data structure which maps this comment better than an int?